container/rkt/client.go

```diff
@@ -38,17 +38,19 @@ var (
 
 func Client() (rktapi.PublicAPIClient, error) {
 	once.Do(func() {
-		_, err := net.Dial("tcp", "localhost:15441")
+		conn, err := net.DialTimeout("tcp", defaultRktAPIServiceAddr, 2*time.Second)
```

Comment:
Any reason you chose a different timeout from the one below? If not, can you create a timeout const to use in both places instead.